helm/chart-operator-chart/templates/rbac.yaml

```diff
@@ -102,6 +102,7 @@ rules:
   verbs:
   - "create"
   - "get"
+  - "update"
```

Comment:
Adding rbac due to an error: `W 06/12 12:23:59 retrying backoff in '6.632270629s' due to error | backoff/notifier.go:13
	/go/src/github.com/giantswarm/chart-operator/vendor/github.com/giantswarm/helmclient/ensure_tiller_installed.go:261:
	deployments.extensions "tiller-deploy" is forbidden: User "system:serviceaccount:giantswarm:chart-operator" cannot update resource "deployments" in API group "extensions" in the namespace "giantswarm"`